docs/README.md

```diff
@@ -868,6 +872,274 @@ In this setup there is a Primary cluster (`cluster1`) and a Remote cluster (`clu
     kubectl delete ns sample --context="${CTX_CLUSTER2}"
     ```
 
+### External-Controlplane
+
+These instructions install an [external-controlplane](https://istio.io/latest/docs/setup/install/external-controlplane/) Istio deployment using the Sail Operator and Sail CRDs. **Before you begin**, ensure you complete the [common setup](#common-setup).
```

Comment:
This might not be obvious but it is part of the pre-req instructions: https://github.com/istio-ecosystem/sail-operator/tree/main/docs#multi-cluster:

> Each deployment model requires you to install the Sail Operator and the Sail CRDs to every cluster that is part of the mesh.

I think that should just be another bullet point in the list.